### PR TITLE
Scenario created for Search Contact

### PR DESCRIPTION
**Pull request by ritaadhikari**
### description
Feature file searchContact.feature for search contact feature is created and different scenarios for the search feature has been created.

### issues
https://github.com/SagarGi/FeatureFiles/issues/7

### CODE REVIEW

File: Features2/tests/features/searchContact.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        | abcde | abcde@gmail.com | 00000000 | Professional |
+        | xyz | xyz@gmail.com | 111111111 | Personal |
+        | example |example@gmail.com | 22222222222| Professional |
+        | example2 | example2@gmail.com |3333333333|Personal|
+        | abcd1 |sabhdb@gmail.com | 9876666 | Personal |
+        |  cvbn |cvbn@gmail.com | 090099090 | Professional |
+        | mnbv | mnvb@gmail.com | 098876677 | Professional |
+
+     Scenario:Search contact by email
+        When the user enters the letter of email in the search icon
```
Comment (SagarGi) [inline Features2/tests/features/searchContact.feature]: i think you have to pass some value to search contact and get the required result right? so pass the value.

File: Features2/tests/features/searchContact.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        Then  it shows all the contacts started from the entered email
+
+
+    Scenario:Search contact by number
+         When the user enters the  number in the search icon
```
Comment (SagarGi) [inline Features2/tests/features/searchContact.feature]: pass number value

File: Features2/tests/features/searchContact.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        When the user enters the letter of email in the search icon
+        Then  it shows all the contacts started from the entered email
+
+
+    Scenario:Search contact by number
+         When the user enters the  number in the search icon 
+         Then all the contacts in which Phone starts with the given number will shwon
+      
+    Scenario:Search with name
+         When the user enters the name
```
Comment (SagarGi) [inline Features2/tests/features/searchContact.feature]: pass name value

File: Features2/tests/features/searchContact.feature [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    Background:
+     Given the user has added following contacts
+        | name | Email | Phone | Contact Type |
+        | abcde | abcde@gmail.com | 00000000 | Professional |
+        | xyz | xyz@gmail.com | 111111111 | Personal |
+        | example |example@gmail.com | 22222222222| Professional |
+        | example2 | example2@gmail.com |3333333333|Personal|
+        | abcd1 |sabhdb@gmail.com | 9876666 | Personal |
+        |  cvbn |cvbn@gmail.com | 090099090 | Professional |
+        | mnbv | mnvb@gmail.com | 098876677 | Professional |
```
Comment (kiranadh1452) [inline Features2/tests/features/searchContact.feature]: Use proper formatting.

File: Features2/tests/features/searchContact.feature [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+     Scenario:Search contact by email
+        When the user enters the letter of email  with "email"in the search icon
+        Then  it shows all the contacts started from the entered email
```
Comment (kiranadh1452) [inline Features2/tests/features/searchContact.feature]: Is the value of email field "email" here ?

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features2/tests/features/searchContact.feature (hunk 5 above)
Comment (ritaadhikari) [reply Features2/tests/features/searchContact.feature]: corrected

File: Features2/tests/features/searchContact.feature [hunk 6]
```diff
@@ -0,0 +1,30 @@
+Feature:Search contact
+    As a user
+    I want to search the contact from the contact list
+    So I can easily get the required contact from the list
+    
+    
+    Background:
+    Given the user has navigated to the homepage
+     And the user has added following contacts
```
Comment (kiranadh1452) [inline Features2/tests/features/searchContact.feature]:
```suggestion
     Given the user has navigated to the homepage
     And the user has added following contacts
```

File: Features2/tests/features/searchContact.feature [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        | abcde    | abcde@gmail.com    | 00000000   | Professional |
+        | xyz      | xyz@gmail.com      | 111111111  | Personal     |
+        | example  |example@gmail.com   | 22222222222| Professional |
+        | example2 | example2@gmail.com | 333333333  | Personal     |
+        | abcd1    |sabhdb@gmail.com    | 9876666    | Personal     |
+        |  cvbn    |cvbn@gmail.com      | 090099090  | Professional |
+        | mnbv     | mnvb@gmail.com     | 098876677  | Professional |
+ 
+     Scenario:Search contact by email
+        When the user enters the letter of email  with "abcde@gmail.com" in the search icon
```
Comment (kiranadh1452) [inline Features2/tests/features/searchContact.feature]:
```suggestion
        When the user enters "abcde@gmail.com" in the search icon
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]
Reuse the search steps.

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**